site/gatsby-site/src/components/submissions/SubmissionEditForm.js

```diff
@@ -57,6 +57,8 @@ const SubmissionEditForm = ({ handleSubmit, saving, setSaving, userLoading, user
 
   const { i18n, t } = useTranslation(['submitted']);
 
+  const { t: tValidation } = useTranslation(['validation']);
```

Comment:
There is no need to define a new `t` instance for each translation module.
Please change this to a single statement in the `line 58`:

```
const { i18n, t } = useTranslation(['submitted', 'validation']);
```